tests/integration_tests/fixtures/energy_dashboard.py

```diff
@@ -70,30 +69,31 @@ def _get_dataframe():
 
 
 def _create_energy_table() -> list[Slice]:
-    table = create_table_metadata(
-        table_name=ENERGY_USAGE_TBL_NAME,
-        database=get_example_database(),
-        table_description="Energy consumption",
-    )
-    table.fetch_metadata()
-
-    if not any(col.metric_name == "sum__value" for col in table.metrics):
-        col = str(column("value").compile(db.engine))
-        table.metrics.append(
-            SqlMetric(metric_name="sum__value", expression=f"SUM({col})")
-        )
-    table.fetch_metadata()
-
-    slices = []
-    for slice_data in _get_energy_slices():
-        slice = _create_and_commit_energy_slice(
-            table,
-            slice_data["slice_title"],
-            slice_data["viz_type"],
-            slice_data["params"],
+    with app.app_context():
```

Comment:
Can't use the `app_context` fixture due to scoping, i.e., this is invoked by the `load_energy_table_data` fixture which is scoped at the session level.